zombies.js

```diff
@@ -287,20 +397,26 @@
  * @param {number} speed            The zombie's speed.
  */
 
+function RangedZombie(health, strength, speed) {
+  Zombie.call(this, health, strength, speed);
+}
+RangedZombie.prototype = Object.create(Zombie.prototype, {
+  constructor: {
+    value: Zombie
+  }
+});
```

Comment:
This should follow structure: `DeathMetalSong.prototype = Object.create(MetalSong.prototype);`